source/reference/known-issues.txt

```diff
@@ -68,3 +68,23 @@ Given the exact same |sql| query, the |bi-short| does not return
       GROUP BY class;
       
       Empty set (0.00 sec)
+
+
+Mapping Potentially Null Values at Different Object Depths
+----------------------------------------------------------
+A known mapping error can occur that causes all returned values to be ``NULL`` 
+in the result. This error occurs when you query a collection in which documents
```

Comment:
A few small suggestions here
- can we use active voice here?
-  it's implied that we're referring to the documents in the collection, so we can remove that bit of extra language
- If you decide to change the title to be more focused on the error itself, we can maybe include that high-level language about object depths

Maybe something like:

> 
> A known mapping error can occur that causes |bi-short| to return only ``NULL``
> values. This error occurs when you map potentially ``NULL`` values at
> different object depths. Specifically, the error occurs when you query a 
> collection that contains.... etc